assets/components/ctaLink/ctaLink.jsx

```diff
@@ -20,18 +22,24 @@ type PropTypes = {
 // ----- Component ----- //
 
 export default function CtaLink(props: PropTypes) {
+  const accessibilityHintId = props.id ? `accessibility-hint-${props.id}` : UUID();
+  const ctaUniqueClassName = `component-cta-link ${props.ctaId}`;
   return (
-    <a
-      id={props.id}
-      className="component-cta-link"
-      href={props.url}
-      onClick={props.onClick}
-      onKeyPress={props.onClick ? clickSubstituteKeyPressHandler(props.onClick) : null}
-      tabIndex={props.tabIndex}
-    >
-      <span>{props.text}</span>
-      <Svg svgName="arrow-right-straight" />
-    </a>
+    <span>
+      <a
+        id={props.id}
+        className={ctaUniqueClassName}
+        href={props.url}
+        onClick={props.onClick}
+        onKeyPress={props.onClick ? clickSubstituteKeyPressHandler(props.onClick) : null}
+        tabIndex={props.tabIndex}
+        aria-describedby={accessibilityHintId}
+      >
+        <span>{props.text}</span>
+        <Svg svgName="arrow-right-straight" />
+      </a>
+      <div id={accessibilityHintId} className="accessibility-hint">{props.accessibilityHint}</div>
```

Comment:
A `<p>` makes more sense here, as it describe text

Reply:
Even though it's positioned off-screen anyway? Does a `<p>` change the way assistive tech interprets the content? Just curious.

Reply:
It depends on the tech, but no, probably not. It's just that `<div>` doesn't mean anything whereas `<p>` means text.

Reply:
Ok, I probably misunderstood the nature of `aria-describedby` elements. I'll change them.